diff -u -r a/include/uapi/linux/elf-em.h b/include/uapi/linux/elf-em.h
--- a/include/uapi/linux/elf-em.h	2019-03-03 23:21:29.000000000 +0000
+++ b/include/uapi/linux/elf-em.h	2019-03-08 20:57:35.153998000 +0000
@@ -46,6 +46,7 @@
 #define EM_BPF		247	/* Linux BPF - in-kernel virtual machine */
 #define EM_CSKY		252	/* C-SKY */
 #define EM_FRV		0x5441	/* Fujitsu FR-V */
+#define EM_AVR32	185	/* Atmel AVR32 */
 
 /*
  * This is an interim value that we will use until the committee comes
