--- a/include/uapi/linux/if_arp.h	2017-04-30 19:47:48.000000000 -0700
+++ b/include/uapi/linux/if_arp.h	2017-06-03 08:06:55.000000000 -0700
@@ -23,6 +23,9 @@
 #ifndef _UAPI_LINUX_IF_ARP_H
 #define _UAPI_LINUX_IF_ARP_H
 
+#ifndef _NET_IF_ARP_H
+#define _NET_IF_ARP_H
+
 #include <linux/netdevice.h>
 
 /* ARP protocol HARDWARE identifiers. */
@@ -157,5 +160,6 @@
 
 };
 
+#endif /* _NET_IF_ARP_H */
 
 #endif /* _UAPI_LINUX_IF_ARP_H */
